chromium_src/chrome/browser/ui/webui/favicon_source.cc

```diff
@@ -6,13 +6,59 @@
 #include "build/build_config.h"
 
 #if defined(OS_ANDROID)
+#include "extensions/buildflags/buildflags.h"
 #include "ui/native_theme/native_theme.h"
 
 #define GetInstanceForNativeUi BraveGetInstanceForNativeUi
 #define IDR_DEFAULT_FAVICON_32 IDR_DEFAULT_FAVICON
 #define IDR_DEFAULT_FAVICON_64 IDR_DEFAULT_FAVICON
 #define IDR_DEFAULT_FAVICON_DARK_32 IDR_DEFAULT_FAVICON_DARK
 #define IDR_DEFAULT_FAVICON_DARK_64 IDR_DEFAULT_FAVICON_DARK
-#endif
+#if !BUILDFLAG(ENABLE_EXTENSIONS)
+// Exclude extension headers to avoid build errors
+#define EXTENSIONS_BROWSER_EXTENSION_REGISTRY_H_
```

Comment:
this is a good solution, better than the one I suggested

Reply:
cc: @samartnik